minard/templates/channel_status.html

```diff
@@ -324,6 +324,11 @@ <h3>History of FEC and/or DB Swaps</h3>
 			<td>{{ row['timestamp'].strftime("%Y-%m-%d") }}</td>
 			<td><a href="http://snopl.us/debugdb/board/{{ '%04x' % row['mbid'] }}">{{ '0x%04x' % row['mbid'] }}</a></td>
 			<td><a href="http://snopl.us/debugdb/board/{{ '%04x' % row['dbid'] }}">{{ '0x%04x' % row['dbid'] }}</a></td>
+                        {% if test_failed and loop.first %}
+                            <td> {{ test_failed }} </td>
+                        {% else %}
+                            <td> </td>
+                        {% endif %}
```

Comment:
Should this just be it's own column? How do you match up the ECAL test that failed with the time it was switched?